Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -188,7 +188,7 @@ void GameCubePane::OnConfigPressed(int slot)
   {
     bool ok;
     const auto new_mac = QInputDialog::getText(
-        this, tr("Broadband adapter MAC address"), tr("Enter new broadband adapter MAC address:"),
+        this, tr("Broadband Adapter MAC address"), tr("Enter new broadband adapter MAC address:"),
```

Comment:
I capitalized that first one because it's a window title and we (mostly) capitalize every word in window titles. But because neither broadband nor adapter are proper nouns, in a sentence "broadband adapter" wouldn't be capitalized.

Reply:
Well, in this case it's probably best to capitalize it in both places, since what it is referring to in this case is the name of a piece of hardware that Dolphin is emulating - the GameCube Broadband Adapter.

Reply:
Ohh, I didn’t know it was a specific piece of hardware! That makes more sense, will fix it.